### PR TITLE
Add review form

### PR DESCRIPTION
**Pull request by PBandJaimie**
@a-nouraldean @Taniel-P @EJLEE808 

This is in place of my botched PR request from yesterday!

"Hi friends! I have almost completed all form functionality. Probably the most difficult thing I've ever coded. I still need to figure out how to get photos to upload with thumbnail previews and then need to get a post method working on submit."

### CODE REVIEW

File: client/dist/styles.css [hunk 1]
```diff
@@ -359,4 +324,113 @@ div {
...
+
+.formStars {
+  display: inline-flex;
+  width: 11em;
+  justify-content: space-between;
+}
+
+.meaning {
+  padding-top: 0.45em;
 }
```
Comment (a-nouraldean) [inline client/dist/styles.css:436]: Nice updates to your css

File: client/src/components/Reviews/AddReview.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import ReviewForm from './ReviewForm.jsx';
+
+const AddReview = (props) => {
+  const [show, setShow] = useState(false);
```
Comment (a-nouraldean) [inline client/src/components/Reviews/AddReview.jsx:5]: Nice use of React hooks and ES6

File: server/routes/qnaHelper 2.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        Authorization: config.TOKEN
+      }
+    })
+
+    return response.data;
+
+  } catch (error) {
+    console.log('Error in Ansyc Await: ', error);
+  }
+};
```
Comment (a-nouraldean) [inline server/routes/qnaHelper 2.js:19]: @PBandJaimie It seems your branch has two qnaHelper files. I think you can delete the second one

**Review by a-nouraldean — CHANGES_REQUESTED** [review]
@PBandJaimie Great job getting these components done. I see that you have a second qnaHelper file called "naHelper 2.js" which I do not think you need. After good to go.

File: server/routes/qnaHelper 2.js (hunk 3 above)
Comment (PBandJaimie) [reply server/routes/qnaHelper 2.js:19]: Strange! But I delete the second one. Done!!!

**Review by a-nouraldean — APPROVED** [review]
Thanks! This looks good!

**Comment by a-nouraldean** [comment]
Keep it up! Let's race to the finish line.

**Pull request merged**